index.html

```diff
@@ -32,5 +35,182 @@ <h3>LET'S CONNECT</h3>
+                </div>
+                <button class="resume button">Get my resume</button>
+            </div>
+            <div class="skill_list">
+                <div class="pgrm_languages">
+                    <div class="lang_head">
+                        <img class="lang_head_icon" src="downarrow.svg">
+                        <h6>Languages</h6>
+                    </div>
+                    <ul class="languange_list">
+                        <li class="lang_item">
+                            <p class="langs">Javascript</p>
+                            <img class="langs_img" id="js" src="js.svg">
+                        </li>
+                        <li class="lang_item">
+                            <p class="langs">HTML</p>
+                            <img class="langs_img" id="ht" src="html.svg">
+                        </li>
+                        <li class="lang_item">
+                            <p class="langs">CSS</p>
+                            <img class="langs_img" id="cs" src="css.svg">
+                        </li>
+                    </ul>
+                    <div class="containers">
+                        <span class="text">Frameworks</span>
+                        <img src="sidearrow.svg" alt="Your Image" class="image">
+                    </div>
+                    <img class="line" src="line.svg">
+                    <div class="containers">
+                        <span class="text">Skills</span>
+                        <img src="sidearrow.svg" alt="Your Image" class="image">
+                    </div>
+                    <img class="line" src="line.svg">
+                </div>
+            </div>
+        </section>
+
+
+        <section class="contact-form" id="cont-form">
+            <div >
+                <div class="contain">
+                    <h1>Contact Me</h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature that you need built,
+                        or a project that needs coding, I'd love to help with it!
+                    </p>
+                    <form id="fs-frm" class="fs" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xoqzownz" method="post">
+                        <fieldset>
+
+                        <input type="text" name="name" id="full-name" placeholder="Your Name" maxlength="30" required>
+
+                        <input type="email" name="_replyto" id="email-address" placeholder="email@example.com" required>
+
+                        <textarea name="message" id="message" rows="5" placeholder="Write your message here" maxlength="500" required></textarea>
+
+                        <input id="sub" class="button" type="submit" value="Get in touch" required>
+                        </fieldset>
+                    </form>
+                    <img class="shape" src="Shape.svg">
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] As requested by the previous reviewer kindly consider sticking to the Figma design for all the sections on your page mainly this contact section because currently everything is messed up the background image is totally wrong and even the other section cause working on the contact me part doesn't mean that your other parts have to be messed up so fixing that will make your website look as an exact duplicate of the provided design and that will make your work professional check the images below fro more clarrification
- your design 👇 
<img width="289" alt="Screen Shot 2023-06-07 at 06 38 37" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/60146030/6f75ba2d-2f43-43bc-9258-18bd2fecf00b">

- Figma design 👇 
<img width="287" alt="Screen Shot 2023-06-07 at 06 41 13" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/60146030/860b9e11-0881-46cc-8232-f4e6e90b999b">

Reply:
Hello @divinecharlotte my review is still pending and I've made the required changes. Can you kindly activate it so it turns to required changes for me to be able to submit my project.